F1/F1/Results/ResultsViewModel.swift

```diff
@@ -19,12 +21,23 @@ class ResultsViewModel {
         self.delegate = delegate
     }
 
+    // MARK: Computed
     var allResultsCount: Int {
         return allResults?.count ?? 0
     }
 
+    var raceResultsCount: Int {
+        raceResult?.count ?? 0
+    }
+
+    var raceName: String {
+        race?.raceName ?? "Race Name"
+
+    }
+
+    // MARK: Functions
     func allResult(atIndex: Int) -> Race? {
-        return allResults?[atIndex] ?? nil
+        allResults?[atIndex] ?? nil
```

Comment:
just checking, you do you want to return nil if there is nothing there?

Reply:
Yes I do 🙈